pkg/golinters/stylecheck/testdata/fix/in/stylecheck.go

```diff
@@ -1,3 +1,5 @@
+// Package testdata ...
```

Comment:
why we cannot just exclude meaningless dirs?

Reply:
Here, we are inside a linter test, so this is expected.